templates/incl-editor-and-output.html

```diff
@@ -133,6 +133,12 @@
             </div>
           </form>
         </div>
+        <div class="pygame-modal py-2 text-left px-3 border-4 border-teal-500 mt-3 rounded bg-green-100" style="display: none">
+          <!--Title-->
+          <div class="flex justify-between items-center">
+            <p class="text-2xl font-bold caption">{{_('pygame_waiting_for_input')}}...</p>
```

Comment:
Can we add the `...` to the translation key as well? It might be the case that in some languages the ... are not the standard for "waiting...". If we add it to the translation we give contributors the freedom to decide what suits best in their language.